app/models/floating_ip.rb

```diff
@@ -18,4 +19,9 @@ def self.available
   def name
     address
   end
+
+  def self.class_by_ems(ext_management_system)
```

Comment:
@durandom ,

this is called from the controller which is coming in separate PR with the UI. I moved it there, thanks